src/app/pages/vm/vm-list/vm-list.component.ts

```diff
@@ -251,7 +251,39 @@ export class VMListComponent {
             icon: "delete",
             label: T("Delete"),
             onClick: delete_row => {
-                this.entityList.doDelete(delete_row);
+                const parent = this;
+                const conf: DialogFormConfiguration = {
+                    title: T('Delete Virtual Machine'),
+                    fieldConfig: [
+                        {
+                            type: 'checkbox',
+                            name: 'zvols',
+                            placeholder: T('Delete data too?'),
+                            value: false
+                        },
+                        {
+                            type: 'checkbox',
+                            name: 'force',
+                            placeholder: T('Force delete?'),
+                            value: false,
+                            tooltip: T('When selected, the system will attempt to delete this Virtual Machine even if \
```

Comment:
Here's a suggestion for a simplified help text that clearly defines ON and OFF behaviors for the checkbox:
```
                            tooltip: T('Set to ignore the Virtual \
 Machine status during the delete operation. Unset to prevent deleting \
 the Virtual Machine when it is still active or has an undefined state.')
```